packages/terra-toggle/src/Toggle.jsx

```diff
@@ -39,7 +41,7 @@ class Toggle extends React.Component {
     // within the toggle to disable users from focusing to them
     if (!this.props.isOpen) {
       if (this.contentContainer) {
-        this.contentContainer.classList.add('is-closed');
+        this.contentContainer.setAttribute('data-closed', 'true');
```

Comment:
Cleaned this up some here: https://github.com/cerner/terra-core/pull/589/commits/bc6407c59c4122545ad1379e562f5c2b24bb3400